src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -239,6 +257,34 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
     if not os.path.exists(os.path.join(base_dir, subset_dir)):
         os.mkdir(os.path.join(base_dir, subset_dir))
 
+    # define path to channel normalization values
+    channel_norm_path = os.path.join(
+        base_dir, pixel_output_dir, 'channel_norm.feather'
```

Comment:
To be like super explicit, can we change this to `channel_norm_pre_rowsum.feather`? To match the corresponding `channel_norm_post_rowsum` file name.

Reply:
Wait actually I just realized the default in the function call to `create_pixel_matrix` is `channel_norm_post_rowsum.feather` - can we change to `channel_norm_post_rownorm.feather`?

Reply:
I'm a little confused: since we're updating this section: https://github.com/angelolab/ark-analysis/blob/3fe18bb7ee8e2b83402e67055e82533507deb5ba/src/ark/phenotyping/pixie_preprocessing.py#L331-L345, to use `channel_percentile_prenorm`, would `channel_norm_path` not contain the pre rowsum norm values? Or am I misunderstanding something?

Reply:
Actually, I think it would be better to move `channel_norm.feather` (to be changed to `channel_norm_pre_rownorm.feather`) to be an input variable in the function call to `create_pixel_matrix`. Since this exists: `norm_vals_name='channel_norm_post_rowsum.feather'`, I think it would be better to make both input variables. So it would look something like this: 
```
def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
                        img_sub_folder="TIFs", seg_suffix='_whole_cell.tiff',
                        pixel_output_dir='pixel_output_dir',
                        data_dir='pixel_mat_data',
                        subset_dir='pixel_mat_subsetted',
                        is_mibitiff=False,
                        blur_factor=2, subset_proportion=0.1, seed=42
                        norm_vals_name_pre_rownorm='channel_norm_pre_rownorm.feather', #see this change
                        norm_vals_name_post_rownorm='channel_norm_post_rownorm.feather', #see this change
                        channel_percentile_pre_rownorm=0.99, #see this change
                        channel_percentile_post_rownorm = 0.999, #see this change
                        multiprocess=False, batch_size=5):
```